RELEASE_NOTES.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [0.11.7] - TBD
+- The internal method `save_genome_mass` was added to facilitate the batch saving of genomes
```

Comment:
No need to add release notes for internals, just say that genomes are now saved in batches to the workspace.

Reply:
👍

Reply:
You should add the new method name though. No other methods had changed behavior right?

Reply:
Which new method name are you referring to?

Reply:
Whatever the new method was that was added to the spec in 0.11.7

Reply:
I added save_genome_mass at the beginning. You replied with [this link](https://github.com/kbaseapps/GenomeFileUtil/pull/212#discussion_r1722491038). 

Are you referring to `set_up_single_params` and `validate_mass_params`, or do you mean something else?
These methods already existed before.

Reply:
The release notes there referred to an internal method, which don't need to be added to release notes. Public methods should be added to the release notes

Reply:
The following functions have changed from private to public:

`set_up_single_params`
`validate_mass_params`
`get_int`
`check_dna_sequence_in_features`

I will add these changes to the release notes then.

Reply:
Those aren't in the public API

Reply:
Only functions in the spec

Reply:
E.g. only functions that someone using GFU in another SDK module would care about

Reply:
I see the reason for confusion, added an edit to my message above

Reply:
Nothing has been added to the specification.

Reply:
https://github.com/kbaseapps/GenomeFileUtil/compare/bee2c55...master

Reply:
Meh, links to files in the diff don't seem to work

Reply:
I assume we are all set and there are no further issues to address regarding this conversation?

Reply:
Is the new API function in the release notes?

Reply:
It is:

```
- Genomes are now saved in batches to the workspace
- Unusable `export_genome_features_protein_to_fasta` function was removed
- The `genbanks_to_genomes` method was added to allow users to upload multiple
genome objects at once
```

Does the 1st line apply to any functions other than `genbanks_to_genomes`? if not it can just be removed

Reply:
👍

Reply:
:+1: